### PR TITLE
Change to Signed Statements for Hashed Payloads

### PR DESCRIPTION
**Pull request by SteveLasker**

### CODE REVIEW

**Comment by SteveLasker** [comment]
Test of these updates: https://github.com/synsation-corp/synsation-web/actions/runs/9980507837

File: scitt-scripts/entrypoint.sh [hunk 1]
```diff
@@ -1,49 +1,92 @@
...
+  --subject $SUBJECT \
   --output-file $SIGNED_STATEMENT_FILE
 
-echo "SCITT Register to https://app.datatrails.ai/archivist/v1/publicscitt/entries"
+if [ ! -f $SIGNED_STATEMENT_FILE ]; then
+  echo "ERROR: Signed Statement: [$SIGNED_STATEMENT_FILE] Not found!"
+  exit 126
+fi
+
+echo "Register the SCITT SIgned Statement to https://app.datatrails.ai/archivist/v1/publicscitt/entries"
```
Comment (CBonnell) [inline scitt-scripts/entrypoint.sh]:
```suggestion
echo "Register the SCITT Signed Statement to https://app.datatrails.ai/archivist/v1/publicscitt/entries"
```

File: scitt-scripts/create_signed_statement.py [hunk 2]
```diff
@@ -96,18 +108,25 @@ def create_signed_statement(
...
     }
 
     unprotected_header = {
         X5chain: issuer.x5chain
     }
 
+    # now create a sha256 hash of the payload
+    #
+    # NOTE: any hashing algorithm can be used.
+    payload_hash = sha256(payload.encode("utf-8")).digest()
```
Comment (CBonnell) [inline scitt-scripts/create_signed_statement.py:122]: Future enhancement idea: fetch the payload at the payload location and compare its hash with the specified payload file hash

**Review by CBonnell — APPROVED** [review]
LGTM, thanks @SteveLasker

**Pull request merged**